src/ci/docker/cross/Dockerfile

```diff
@@ -19,7 +19,13 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
   bzip2 \
   patch \
   libssl-dev \
-  pkg-config
+  pkg-config \
+  swig \
+  libedit-dev \
+  libncurses5-dev
+
+RUN curl -L https://cmake.org/files/v3.8/cmake-3.8.0-rc1-Linux-x86_64.tar.gz | \
```

Comment:
CMake 3.8.0 is (going to) be the first version with Fuchsia support, RC1 was only released two weeks ago so I don't think it's available in any Linux distribution just yet. Do you want me to add the comment in this PR?